Functions.Templates/Templates/BlobTrigger-Python/__init__.py

```diff
@@ -0,0 +1,9 @@
+import logging
+
+import azure.functions as azf

```

Comment:
Can we rename `azf` to `func` instead? Most functions users are already familiar with the alias and it's easy to read.